pkg/aggregator/time_sampler.go

```diff
@@ -84,7 +84,7 @@ func (s *TimeSampler) newSketchSeries(ck ckey.ContextKey, points []metrics.Sketc
 	ctx, _ := s.contextResolver.get(ck)
 	ss := metrics.SketchSeries{
 		Name:       ctx.Name,
-		Tags:       ctx.Tags(),
+		Tags:       ctx.Tags().ToNewSliceString(),
```

Comment:
We should probably also support this for sketches.

Reply:
I am working on it.

Reply:
Fixed.